tests/sampling/test_jax.py

```diff
@@ -154,6 +154,39 @@ def test_deterministic_samples(sampler):
     assert np.allclose(trace.posterior["b"].values, trace.posterior["a"].values / 2)
 
 
+@pytest.mark.parametrize(
+    "sampler",
+    [
+        # sample_blackjax_nuts,
```

Comment:
this test currently fails for sample_blackjack_nuts

`FAILED tests/sampling/test_jax.py::test_initvals_without_jitter[sample_blackjax_nuts] - TypeError: iteration over a 0-d array`